lib/wireprotocol/write_command.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+const MongoError = require('../error').MongoError;
+const collectionNamespace = require('./shared').collectionNamespace;
+const command = require('./command');
+
+function writeCommand(server, type, opsField, ns, ops, options, callback) {
+  if (ops.length === 0) throw new MongoError('insert must contain at least one document');
+  if (typeof options === 'function') {
+    callback = options;
+    options = {};
```

Comment:
What are these two lines supposed to do?

Reply:
looks like merge pasta 😄 fixed